cegs_portal/static/css/project.css.tw

```diff
@@ -66,32 +66,52 @@
 
   a:visited {
     color: inherit;
+    text-decoration: none;
   }
 
   .th-bg-light-gray {
     background-color: rgb(249, 250, 251);
   }
 
-  table.data-table {
-    border-collapse: collapse;
-    width: 100%;
-    font-family: Helvetica, Roboto, sans-serif;
+  .svg-icon {
+      fill: currentColor;
+      height: 1em;
   }
 
-  table.data-table th {
-    padding: 12px;
-    text-align: left;
-    color: gray;
-    font-weight: bold;
+  .tab-bar {
+      background: #ffffff;
+      border-radius: 10px;
+      padding-left: 10px !important;
+      padding-right: 10px;
+      box-shadow: 0px 2px 5px 4px rgba(0, 0, 0, 0.1);
   }
 
-  table.data-table td {
-    padding: 12px;
-    border-bottom: 1px solid lightgrey;
+  .container {
+    background: #ffffff;
+    box-shadow: 0px 2px 5px 4px rgba(0, 0, 0, 0.1);
+    padding: 20px 30px;
+    border-radius: 10px;
+    margin-bottom: 20px;
+  }
+
+  h1 {
+    font-family: sans-serif;
+    font-size: 2rem;
+    color: #094474;
+    text-shadow: 2px 2px #e3f2fd;
+    margin-bottom: 10px;
+    margin-left: 40px;
+    display: flex;
+    align-items: center;
+  }
+
+  h1 span:nth-child(3),
+  span:nth-child(4) {
+    font-size: 1.5rem;
```

Comment:
What is this for? It seems very specific

Reply:
Header title